bootstrap/central_account_iam/main.tf

```
@@ -4,7 +4,7 @@ locals {
 
 # Role used by Terraform to manage all satellite accounts
 module "gh_oidc_roles" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.18//gh_oidc_role"
+  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v6.1.4"
```

Could you please revert this one back to the original version.  There was a breaking change that came in at `v4.0.0` of the module and it'll be hard to fix:
```suggestion
  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v3.0.18"
```

---

This has been updated! Thank you Pat for your suggestion as I would have broken everything 😬